### PR TITLE
Add new google_chat module

### PR DESCRIPTION
**Pull request by luke-orden**
### What does this PR do?

Add a new `google_chat` module which allows you to send a message to a google chat room.

### What issues does this PR fix or reference?

N/A

### New Behavior
Send a message to a google chat room

### Tests written?

Yes

### Commits signed with GPG?

Yes

### CODE REVIEW

**Review by cachedout — CHANGES_REQUESTED** [review]
Excellent! Before we merge this, would you mind adding information about this new feature to the release notes?

File: salt/modules/google_chat.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    '''
+    Send a message to the google chat room specified in the webhook url.
+
+    .. code-block:: bash
+
+        salt '*' google_chat.send_message "https://chat.googleapis.com/v1/spaces/example_space/messages?key=example_key" "This is a test message"
+    '''
+    headers = {'Content-Type': 'application/json'}
+    data = {'text': message}
+    result = salt.utils.http.query(url,
```
Comment (gtmanfred) [inline salt/modules/google_chat.py]: instead of importing salt.utils.http, please use `__utils__['http.query']()`

**Review by gtmanfred — CHANGES_REQUESTED** [review]
Just one nitpick change.

Otherwise this looks good.

File: salt/modules/google_chat.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+.. versionadded:: Fluorine
+
+To use this module you need to configure a webhook in the google chat room
+where you would like the message to be sent, see:
+
+    https://developers.google.com/hangouts/chat/how-tos/webhooks
+'''
+# Import Python Libs
+from __future__ import absolute_import
```
Comment (rallytime) [inline salt/modules/google_chat.py]: Can you add `print_function` and `unicode_literals` to this `__future__` import? We need that for PY2/PY3 compatibility. :)

**Review by terminalmage — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]
Awesome! :grin:

**Comment by rallytime** [comment]
@loverend If you could add this to the [Fluorine release notes](https://github.com/saltstack/salt/blob/develop/doc/topics/releases/fluorine.rst) we can get this in pending tests. :)

Thanks for this addition!

**Comment by loverend** [comment]
Added to release notes. There wasn't a `new modules` section like in previous release notes, so added this.

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
@loverend Some of your new tests are having some trouble here. Can you take a look?

https://jenkinsci.saltstack.com/job/pr-kitchen-ubuntu1604-py2/job/PR-48919/5/

**Comment by loverend** [comment]
All tests now pass. I have addressed all comments. Please let me know if there are any further changes required.

**Review by gtmanfred — APPROVED** [review]
Awesome! thanks for this! looks great.

**Pull request merged**